crossbeam-utils/src/thread.rs

```diff
@@ -295,7 +295,7 @@ unsafe impl<'scope, T> Send for ScopedJoinHandle<'scope, T> {}
 unsafe impl<'scope, T> Sync for ScopedJoinHandle<'scope, T> {}
 
 /// A handle to a scoped thread
-pub struct ScopedJoinHandle<'scope, T: 'scope> {
+pub struct ScopedJoinHandle<'scope, T> {
```

Comment:
We don't need `T: 'scope` here. Note that [`JoinHandle<T>`](https://doc.rust-lang.org/nightly/std/thread/struct.JoinHandle.html) doesn't require `T: 'static` either.

Reply:
@stjepang I think this new for Rust 2018/NLL, so it might be worth it to leave it to support older Rust versions.